msal/application.py

```diff
@@ -700,6 +700,35 @@ def _validate_ssh_cert_input_data(self, data):
                     "you must include a string parameter named 'key_id' "
                     "which identifies the key in the 'req_cnf' argument.")
 
+    def acquire_token_by_refresh_token(self, refresh_token, scopes):
+        """Acquire token(s) based on a refresh token (RT) obtained from elsewhere.
+
+        In normal MSAL scenarios, you do NOT need to use this method.
```

Comment:
Normal MSAL scenarios does not give a very clear idea on what are these normal scenarios. May be we can write the whole paraphrase like this or any other way to be more clear:
```
You use this method only when you have old RTs from elsewhere,
and now you want to migrate them into MSAL.
Calling this method results in new tokens automatically storing into MSAL

You do NOT need to use this method if you are already using MSAL.
MSAL maintains RT automatically inside its token cache, and an access token can be retrieved
when you call :func:`~acquire_token_silent`.